tests/addons/sinonResponder.js

```diff
@@ -1,34 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-define([
-  'tests/addons/sinon'
-], function (Sinon) {
-
-  return {
-    makeMockResponder: function (requests) {
-      var self = this;
-      var requestIndex = 0;
-
-      return function (returnValue, response) {
-        Promise.resolve().then(function () {
```

Comment:
I couldn't figure out what this `Promise` for was, so I removed it...but something might break in CI to tell me what it was for? :-)